common/persistence/cassandraPersistence.go

```diff
@@ -713,7 +713,6 @@ func (d *cassandraPersistence) CreateWorkflowExecution(request *CreateWorkflowEx
 
 	previous := make(map[string]interface{})
 	applied, iter, err := d.session.MapExecuteBatchCAS(batch, previous)
-	defer iter.Close()
 	if err != nil {
```

Comment:
iter can be leaked in case of err? seems like iter can be valid when there is error in the method MapExecuteBatchCAS().